lib/dialects/mssql/query-interface.js

```diff
@@ -20,47 +22,47 @@
   @param  {Boolean|Function} [options.logging] A function that logs the sql queries, or false for explicitly not logging these queries
  @private
  */
-var removeColumn = function(tableName, attributeName, options) {
-  var self = this;
+const removeColumn = function(tableName, attributeName, options) {
   options = options || {};
+  options = _.assign({ raw: true, logging: options.logging}, options);
```

Comment:
Use `Object.assign`
`logging: options.logging` is redundant
Move the `|| {}` inside the call